### DISCUSSION TITLE
creating new files and folder in file explorer

### DISCUSSION THREAD

**Discussion by abhishekjakhar** [codesandbox/sandpack#687] [Ideas]
Could we create new files and folder on fly, if not then what changes would be required for me to achieve this functionality?

**Comment by danilowoz** [comment]
Currently, SandpackFileExplorer doesn't support it. But I'd recommend using a third package ([sandpack-file-explorer](https://www.npmjs.com/package/sandpack-file-explorer)), which supports this feature and even more.

Let me know if it works for you
